src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -2641,16 +2771,198 @@ public override object PayloadValue(int index)
             switch (index)
             {
                 case 0:
-                    return Count;
+                    return ClrInstanceID;
                 case 1:
+                    return Count;
+                default:
+                    Debug.Assert(false, "Bad field index");
+                    return null;
+            }
+        }
+
+        private int SizeOfGCLOHCompactInfo
+        {
+            get
+            {
+                return PointerSize == 8 ? 32 : 20;
```

Comment:
Can you please add a comment that describes the field layout that is used to generate these values?